.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Get rid of this file entirely. It looks like it's your own VSCode configuration which doesn't need to be stored in the repo.